trade-finance-manager-api/api-tests/v1/amendments/create-amendment-tfm-object-on-amendment-completion.api-test.js

```diff
@@ -293,6 +293,32 @@ describe('update amendment-tfm on amendment completion', () => {
 
         expect(result).toEqual(expected);
       });
+
+      it('should create tfm object excluding facility end date data when the facility is from a non GEF deal type', async () => {
+        const nonGefFacility = { facilitySnapshot: { ...mockFacility.facilitySnapshot, type: FACILITY_TYPE.BOND } };
```

Comment:
FWIW could add a loan test too but all good as is

Reply:
added as I never get to do review comments anymore and i love them